gen/fs_home

```diff
@@ -62,22 +64,34 @@ our $A_F_QUOTAENABLED;             *A_F_QUOTAENABLED =               \'urn:perun
 our $A_GROUP_UNIX_GROUP_NAME;      *A_GROUP_UNIX_GROUP_NAME =        \'urn:perun:group_resource:attribute-def:virt:unixGroupName';
 our $A_GROUP_GROUP_NAME;           *A_GROUP_GROUP_NAME =             \'urn:perun:group:attribute-def:core:name';
 our $A_R_VO_NAME;                  *A_R_VO_NAME =                    \'urn:perun:resource:attribute-def:virt:voShortName';
+our $A_F_NEW_QUOTAS_READY;         *A_F_NEW_QUOTAS_READY =           \'urn:perun:facility:attribute-def:def:readyForNewQuotas';
 
 our $STRUC_GROUPS;   *STRUC_GROUPS =  \"groups";
 
+#headers
+my $dataQuotaHeader = "dataQuota";
+my $dataLimitHeader = "dataLimit";                                                                                                      
+my $fileQuotaHeader = "fileQuota";
+my $fileLimitHeader = "fileLimit";
```

Comment:
Constants are usually written in uppercase.

Reply:
Changed.